modules/core/src/main/java/org/apache/ignite/IgniteSnapshot.java

```diff
@@ -48,4 +50,13 @@
      * @return Future which will be completed when cancel operation finished.
      */
     public IgniteFuture<Void> cancelSnapshot(String name);
+
+    /**
+     * Restore cache group(s) from the snapshot.
```

Comment:
Let's add to the descriptoin that caches that are planning to be restored must be destroyed manually by the user.

Reply:
description changed to
```
Restore cache group(s) from the snapshot.
<p>
<b>NOTE:</b> Cache groups to be restored from the snapshot must not present in the cluster, if they present, they must be destroyed by the user before starting this operation.
```